1.10.5/rhel7/onbuild/Dockerfile

```diff
@@ -19,3 +19,15 @@ LABEL maintainer="Astronomer <humans@astronomer.io>"
 LABEL io.astronomer.docker=true
 LABEL io.astronomer.docker.airflow.onbuild=true
 
+# Install RHEL packages
+ONBUILD COPY packages.txt .
+ONBUILD RUN if [[ -s packages.txt ]]; then \
+    yum update -y && cat packages.txt | xargs yum install -y; \
+  fi
+
+# Install python packages
+ONBUILD COPY requirements.txt .
+ONBUILD RUN pip install --no-cache-dir -q -r requirements.txt
```

Comment:
@ianstanton should we also add a similar `if` guard here too?

Reply:
Updated